dist/spec.json

```diff
@@ -15802,6 +15802,29 @@
       ]
     },
     "ApplicationSettingsApplication": {
+      "properties": {
```

Comment:
Just some notes: I see most of these attributes in the OAS3 spec on SwaApplicationSettingsApplication and SamlApplicationSettingsApplication
but `acsURL` is `acsUrl`. I'm not seeing `orgName`. But there is some drift in the OAS3 spec and some attributes that the monolith at the moment.

I would suggest changing `acsURL` to `acsUrl` so as avoid future compatibility issues when this code is generated with OAS3.